internal/datacoord/services.go

```diff
@@ -159,6 +171,7 @@ func (s *Server) Flush(ctx context.Context, req *datapb.FlushRequest) (*datapb.F
 		TimeOfSeal:      timeOfSeal.Unix(),
 		FlushSegmentIDs: flushSegmentIDs,
 		FlushTs:         ts,
+		ChannelCps:      channelCPs,
```

Comment:
Why channelCPs are needed in FlushResponse?

Reply:
Backup use Flush to get FlushTs. FlushTs is used to mark the end of this backup and used to filtered data > FlushTs. CDC will use channelCPs as the beginning to transport the data in MQ between clusters. So we also support it in Flush